### PR TITLE
Configurable Mesos Role

### PR DESCRIPTION
**Pull request by everpeace**
This is for #46 .
I'm pleased if I could have some feedbacks.

### CODE REVIEW

**Comment by kensipe** [comment]
+1 sgtm

**Comment by mohitsoni** [comment]
+1 SGTM. It would be great if you can document this as part of: https://github.com/mesos/myriad/blob/phase1/docs/myriad-configuration.md, and then we can merge this one.

**Comment by everpeace** [comment]
@mohitsoni Oh, sorry about forgetting updating a document.  I will update the document tonight.

**Comment by everpeace** [comment]
@mohitsoni Hi! I pushed several commits for updating configuration document in each pull request (#51, #52, this)!

File: docs/myriad-configuration.md [hunk 1]
```diff
@@ -23,6 +23,9 @@ frameworkFailoverTimeout: 43200000
 # Myriad's mesos framework name.
 frameworkName: MyriadAlpha
 
+# Myriad's mesos framework role.
+frameworkRole: someRoleName
```
Comment (smarella) [inline docs/myriad-configuration.md:27]: @everpeace I think we should add "frameworkRole" property to https://github.com/mesos/myriad/blob/phase1/src/main/resources/myriad-config-default.yml as well. The value can be left empty.

Comment (everpeace) [inline docs/myriad-configuration.md:27]: @smarella Thank you for good catch.  I'll do that not only for this but also others.

**Pull request merged**